packages/google_maps_flutter/google_maps_flutter_platform_interface/test/types/marker_test.dart

```diff
@@ -201,5 +201,18 @@ void main() {
       expect(marker.zIndexInt, 5);
       expect(marker.zIndex, 5.00);
     });
+
+    test('zIndexInt param copyWith', () {
```

Comment:
This would be better as two tests, one for each version.

Generally speaking, unit tests should be as small as is reasonable. This is conceptually two different tests, one after the other, in the same method.